tests/bugfixes/redmine/test_issue_443.py

```diff
@@ -50,9 +48,8 @@ class CorruptedIopDir(metaclass=system_tests.CaseMeta):
 Exif.Thumbnail.XResolution                   Rational    1  72
 Exif.Thumbnail.YResolution                   Rational    1  72
 Exif.Thumbnail.ResolutionUnit                Short       1  inch
-Exif.Thumbnail.JPEGInterchangeFormat         Long        1  1628
+Exif.Thumbnail.JPEGInterchangeFormat         Long        1  1604
```

Comment:
How is possible that the output of this test has changed ? According to the git history the input file for this test was removed or changed (`$data_path/exiv2-bug443.jpg`).  Was this intentional? I am a bit lost here.